starter-code/css/main.css

```diff
@@ -7,4 +7,98 @@ light blue: #1c5380
 body {
   font: 200 16px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
   margin: 0;
+  background-image : linear-gradient(to bottom, #1c5380 223px, #fff 203px);
 }
+
+.container{
+ 
+  display: flex;
+  justify-content: space-around;
+  width: 70%;
+  margin: 30px auto;
```

Comment:
Using `margin: 30px` auto is good regarding the `auto` part, but the 30px could have been achieved in a better way. This is usually how you center things with Flexbox:
```css
container-element {
  display: flex;
  justify-content: center;
  align-items: center;
} 

child-element {
  width: 70%; /*This is an example, you can give it whatever width, as long as it's defined*/
  height: 200px; /*This is an example, you can give it whatever width, as long as it's defined*/
}
```

So in your case your `<body>` could have been your container, and "container" your `child-element`.